exovetter/odd_even.py

```diff
@@ -56,6 +55,65 @@ def calc_odd_even(time, flux, period, epoch, duration, ingress=None):
     return sigma, odd_depth, even_depth
 
 
+def diagnostic_plot(time, flux, period, epoch,
+                    duration, odd_depth, even_depth):  # pragma: no cover
+    """
+    Parameters
+    ----------
+    time : array
+        Time aray
+    flux : array
+        Array of detrended flux
+    period : float
+        period in units of time
+    epoch : float
+        time of transit in units of time
+    duration : float
+        duration of transit in units of time
+    odd_depth : float
+        odd depth in units of flux
+    even_depth : float
+        even depth in units of flux
+
+    Returns
+    -------
+    None.
+
```

Comment:
"Returns" isn't necessary if it returns nothing.
```suggestion
```

Reply:
good point... the editor just puts that in by default and I forgot to take it out.